proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,118 @@
+
+Questions and proposals for changes to governance are posted as issues in the kubeflow/community repo, and the KSC invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+KSC is composed of 5 (five) members. They are elected according to the election policy [TODO: add link].  
+Seats on the Steering Committee are held by an individual, not by their employer.  
+The current membership of the committee is currently (listed alphabetically by first name):
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+Member</th>
+      <th><br>
+Organization</th>
+      <th><br>
+Github Profile</th>
+      <th><br>
+Term Start</th>
+      <th><br>
+Term End</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Emeritus Committee Members
+
+[This section will be populated when there are retired committee members.]
+
+## Decision process
+
+The steering committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing written policy, creating, removing, or modifying a working group; all spending, hiring, and contracting decisions; official responses to publicly raised issues; or any other decisions that at least half of the members (rounded down) present decide require a vote.
+
+Decisions are made in meetings when a quorum of the members are present and may pass with at least half the members (rounded up) of the committee supporting it.
+
+Quorum is considered reached when at least half of the members (rounded up) are present.  
+Members of KSC may abstain from a vote. Abstaining members will only be considered as contributing to quorum, in the event that a vote is called in a meeting.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the steering committee at [TBD]. This is a private discussion list to which all members of the committee have access.
+1. Open an issue on a kubeflow/community repository and indicate that you would like attention from the steering committee using github tag [TBD].
+
+## Changes to the charter
+
+Changes to the KSC charter may be proposed via a PR on the charter itself. Amendments are accepted with majority consent of the committee as per the decision process outlined above.
```

Comment:
I think this should follow the rule I proposed above (in the "Decision process") section, where 70% of the current KSC members must explicitly vote in support of such amendments to be adopted.

Reply:
Updated to follow "Special Decision Process".